packages/bot-web-ui/src/stores/data-collection-store.js

```diff
@@ -69,7 +73,7 @@ export default class DataCollectionStore {
                     },
                 };
             };
-
+            this.setRunId(this.getHash(this.getXmlHash() + this.root_store.core.client.loginid + Math.random()));
```

Comment:
- Do we need to hash the XML string twice? `this.getXmlHash` suggests that the XML is already hashed.
- It looks like this breaks the flow of logging. Previously a `run_id` would only be set for each individual run of the bot (allowing us to group transactions and runs together), but with this change a new `run_id` is generated for each transaction. Is this intended?

Reply:
no, just wanted to resolve the emptiness of `run_id` which resulted in the error.